sdk/cosmos/azure-cosmos-examples/src/main/java/com/azure/cosmos/examples/BasicDemo.java

```diff
@@ -5,14 +5,14 @@
 import com.azure.cosmos.CosmosAsyncClient;
 import com.azure.cosmos.CosmosAsyncContainer;
 import com.azure.cosmos.CosmosAsyncDatabase;
-import com.azure.cosmos.CosmosAsyncItemResponse;
+import com.azure.cosmos.model.CosmosAsyncItemResponse;
 import com.azure.cosmos.CosmosClientBuilder;
 import com.azure.cosmos.CosmosClientException;
-import com.azure.cosmos.CosmosContainerProperties;
-import com.azure.cosmos.CosmosPagedFlux;
-import com.azure.cosmos.FeedOptions;
-import com.azure.cosmos.FeedResponse;
-import com.azure.cosmos.PartitionKey;
+import com.azure.cosmos.model.CosmosContainerProperties;
+import com.azure.cosmos.model.CosmosPagedFlux;
```

Comment:
I am not sure if CosmosPageFlux is a model or not.

From one perspective  using CosmosPageFlux internally will issue IO request to fetch more results, so it shouldn't be a model.

from another perspective, other query result classes are model, so one may say this one is a model too.

Can we check what some other services are doing? is any other service sdk using pageflux?

Reply:
Sure, to be consistent with other azure SDKs, I reverted the change and moved both CosmosPagedFlux and CosmosPagedIterable back to cosmos package.